### DISCUSSION TITLE
Wizard Step depend on Select Value Causing Problem

### DISCUSSION THREAD

**Discussion by EG-Mohamed** [filamentphp/filament#14680] [Help]
### Package

Panel builder

### Package Version

v3.2.117

### How can we help you?

I have `Wizard` That has One Step and this step includes `ToggleButton` to select I have another Step that is only visible with one of the select values using `->visible()` method. " for sure select is `live()` "

the problem is the function of the visible step works fine but there's no button to go to the next step.

![image](https://github.com/user-attachments/assets/d2a6894d-93b3-483f-9f78-d688cc4bd8c8)
![image](https://github.com/user-attachments/assets/7ed4a61c-881f-4f2f-ad80-ee13c65701fe)

**Comment by leandrocfe** [comment]
Could you share the code please?

**Comment by EG-Mohamed** [comment]
Here's Simple Example
```php
          Wizard::make([
              Wizard\Step::make('First Step')->schema([
                  ToggleButtons::make('check_if')->boolean()->live()
              ]),
              Wizard\Step::make('only_if_true')->schema([
                  TextInput::make('other_field'),
              ])->visible(fn(Get $get) => $get('check_if')),
          ]),
```

![image](https://github.com/user-attachments/assets/32ee1e2c-ab98-4ab1-879c-82f96e30e2b5)
![image](https://github.com/user-attachments/assets/54d413c7-19d3-477e-9d2b-3ba5d4364d42)

**Reply by leandrocfe** [reply]
are you using wizard in Filament Resource or is it a custom page/custom form?

**Reply by EG-Mohamed** [reply]
Filament Resource.

**Reply by leandrocfe** [reply]
If you are using a Resource,  you should use [this trait](https://filamentphp.com/docs/3.x/panels/resources/creating-records/#using-a-wizard)

**Reply by EG-Mohamed** [reply]
Same Problem, there's a step but the next button does not appear.

![image](https://github.com/user-attachments/assets/a5aa391d-de99-407c-a63c-0ff2d8ec3a01)

**Reply by leandrocfe** [reply]
I see the issue. I’ve tried some code, but I don’t have a solution for it at the moment...

**Reply by EG-Mohamed** [reply]
Thank you, mate, I have opened a new issue  

https://github.com/filamentphp/filament/issues/14694
